helpers.ts

```diff
@@ -0,0 +1,12 @@
+import {User} from "./interfaces";
+
+const getHighestFromArr = (arr: Array<number>) => {
+    if (arr.length === 0) {
+        return 0;
+    }
+    return Math.max(...arr) + 1;
+};
+
+const checkUser = (checkedUser: User, users: Object) => Object.entries(users).some( l => l[1].id === checkedUser.id);
```

Comment:
why is users an Object?

Reply:
because it is? that's jsut the way i did things

Reply:
it's not